docs/deploy-operator.md

```diff
@@ -29,10 +29,11 @@ For **testing purposes** you may use [minio](https://github.com/minio/minio) as
 
 ```shell
 kubectl create namespace observatorium-minio
```

Comment:
There is one more link that incorrectly references `master` instead of `main` on line 150, I think it would be good to fix that line as well in one go.